kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -611,12 +613,25 @@ define([
          * Set up event handlers
          */
         setUpEventHandlers() {
+            const self = this;
             this.jobManager.addEventHandler('modelUpdate', {
                 dropdown: () => {
                     this.dropdownWidget.updateState();
                 },
                 jobStatusTable_status: (_, jobArray) => {
-                    jobArray.forEach((jobState) => this.updateJobStatusInTable(jobState));
+                    jobArray.forEach((jobState) =>
+                        this.updateJobStatusInTable.bind(self)(jobState)
+                    );
+                },
+            });
+            this.jobManager.addEventHandler('job-error', {
+                jobStatusTable_error: (_, jobError) => {
+                    this.handleJobError.bind(self)(jobError);
```

Comment:
Before the change in the job manager to replace identically-named handlers, these functions and specifically the `this` context (referring to the first instance of a JobStatusTable) would continue to be executed, long after that job status table had been destroyed.

An alternative would have been to specify the function to be performed and the arguments, which would mean that you could just update the args each time. For the purposes of this bug and the MVP, this is the quickest and easiest solution.